src/transport_performance/urban_centres/raster_uc.py

```diff
@@ -486,12 +497,12 @@ def _get_x_y(
         Parameters
         ----------
         coords : tuple
-            Tuple with coordinates to convert.
-            Must be in format (lat, long) and EPSG: 4326.
+            Tuple with coordinates to convert. Must be in format (lat, long)
```

Comment:
Will remove this requirement from docstring as the tuple can now be of any crs and not necessarily (lat, long).